catalog/plugin/catalog-plugin-metacard-validation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -48,15 +47,13 @@
     <!-- Pre-Query Metacard Validity Checker Plugin -->
     <bean id="pre-query-plugin"
           class="ddf.catalog.metacard.validation.MetacardValidityCheckerPlugin">
-        <!--<cm:managed-properties-->
-        <!--persistent-id="MetacardValidityCheckerPlugin"-->
-        <!--update-strategy="container-managed"/>-->
+        <cm:managed-properties
```

Comment:
It's a bit strange that you have a metatype and a setter, but no `<property.../>`. Shouldn't the property be declared in blueprint as well?